packages/ui/components/unpublished-entity/UnpublishedEntity.tsx

```diff
@@ -13,13 +13,17 @@ export function UnpublishedEntity(props: UnpublishedEntityProps) {
   return (
     <div className="m-8 flex items-center justify-center">
       <EmptyScreen
-        avatar={<Avatar alt={slug ?? ""} imageSrc={`/team/${slug}/avatar.png`} size="lg" />}
+        avatar={
+          <Avatar
+            alt={slug ?? ""}
+            imageSrc={props.orgSlug ? `/org/${slug}/avatar.png` : `/team/${slug}/avatar.png`}
+            size="lg"
```

Comment:
both org and org subteams have orgSlug but both have different url for image

Reply:
Should be fixed now, thanks @Udit-takkar